dist/src/main/java/io/camunda/identity/IdentityModuleConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * Entry point for the Identity modules by using the the {@link
+ * io.camunda.application.Profile#IDENTITY} profile, so that the appropriate Identity application
+ * properties are applied.
+ */
+@ComponentScan(
+    basePackages = {"io.camunda.identity", "io.camunda.authentication"},
+    excludeFilters = {
+      @ComponentScan.Filter(
+          type = FilterType.REGEX,
+          pattern = "io\\.camunda\\.identity\\.starter\\..*")
+    })
+@ConfigurationPropertiesScan(basePackages = {"io.camunda.identity", "io.camunda.authentication"})
+@EnableAutoConfiguration
+@Profile("identity")
```

Comment:
The authentication layer will also only be available with profile `identity` right? This should not be the case. So from this point of view it makes sense to have the auth classes being separated from the `identity` profile.

Reply:
That is correct, I added it this way because in my opinion having an auth layer without Identity didn't make sense, without Identity, the auth layer does not have a way to manage local users etc also, we currently have a hard dependency from `identity-backend` to the `auth-starter` because the Identity backend uses Spring security level classes `CamundaUserDetailsManager` for example. If we elect to only start Identity then the startup fails because of the missing bean that is in the `auth-starter` module.

I will work on trying to separate this though